### PR TITLE
- Added numerous compatibility fixes for Doom2: Maps 02 to 11. These …

### PR DESCRIPTION
**Pull request by raa-eruanna**
…fixes mark previously unmarked secrets, and texture fixes in several locations that had HOMs.

### CODE REVIEW

**Comment by raa-eruanna** [comment]
I wasn't sure if you wanted the new secrets. All fixes here come from information obtained from the Doom Wiki. That is why I created a pull request instead of committing directly.

**Comment by alexey-lysiuk** [comment]
I think a new secret is a bit too much for compatibility fix.
Removal of unreachable one is OK but addition of a new secret looks contradictory to me.
Although this doesn't mean that it's useless or doesn't make sense. Let's wait for Graf.

File: wadsrc/static/compatibility.txt [hunk 1]
```diff
@@ -484,6 +484,63 @@ F6EE16F770AD309D608EA0B1F1E249FC // Ultimate Doom, e4m3
...
+{
+	// missing textures
+	setwalltexture 456 back top SUPPORT3
+	setwalltexture 108 front top STONE
+	setwalltexture 109 front top STONE
+	setwalltexture 110 front top STONE
+	setwalltexture 111 front top STONE
+	setwalltexture 127 front top STONE
+	setwalltexture 128 front top STONE
+	// remove erronious blue keycard pickup ambush sector tags (nearby viewing windows, and the lights)
```
Comment (Gaerzi) [inline wadsrc/static/compatibility.txt]: *[erroneous](http://www.tfd.com/erroneous) :stuck_out_tongue:

Comment (raa-eruanna) [inline wadsrc/static/compatibility.txt]: Fixed. :P

**Comment by coelckers** [comment]
No adding unmarked secrets, please. You may add everything else to the repo yourself.

**Pull request closed without merging**